### DISCUSSION TITLE
Как сделать сайт чтобы в телфоне не изменялись фотографии?

### DISCUSSION THREAD

**Discussion by Eldos-Git** [community/community#68511] [Other Feature Feedback, Questions, & Ideas]
### Изменяеться фотографии сайта в телефонном режиме

Question

### Как сделать сайт чтобы в телефоне не изменялись фотографии?

Вот мой сайт: https://[eldos-git.github.io/portfolio/](https://eldos-git.github.io/portfolio/)

В компьютерным режиме все нормально работает но в телефоне совсем другое. Там изменяется фотографии на других вообще. некоторые даже не показывается. Как исправить? Помогите пожалуйста.

![photo_5235572019112823032_y](https://github.com/community/community/assets/68679528/b7a631be-f578-4b4b-8de8-523b3a24865f)

**Comment by queenofcorgis** [comment]
Hi @Eldos-Git , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
